### PR TITLE
update to Contributing file - previewing quickstarts

### PR DESCRIPTION
**Pull request by MelissaFlinn**

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -6,20 +6,20 @@
 
 The guides toolchain is heavily dependent on AsciiDoc and Node.
 
-. Install your favourite tool for editing AsciiDoc
-. Install link:https://nodejs.org/en/about/releases/[Node Active LTS] - support for link:https://github.com/nvm-sh/nvm[Node Version Manager] is present; having installed nvm run
+. Install your favourite tool for editing AsciiDoc (for example, Atom).
+. Install the latest Active long-term support (LTS) version of link:https://nodejs.org/en/about/releases/[Node] (`node`) .
+. Install link:https://github.com/nvm-sh/nvm[Node Version Manager] (`nvm`).
+. After you install `node` and `nvm`, run the following commands from your local  `app-services-guides` repository's top-level directory (for example, `C:/mylocalGitRepos/app-services-guides/`):
```
Comment (bhardesty) [inline CONTRIBUTING.adoc:12]: Might be better to show a Unix style file path in the example.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -107,18 +107,18 @@ A schema for `quickstart.yml` is provided in `quickstart.schema.json`. To load t
 * Follow link:https://www.jetbrains.com/help/idea/json.html#ws_json_schema_add_custom[this guide for IntelliJ], use `**/quickstart.yml` as the file path pattern
 * Install link:https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml[the YAML extension for VS Code], follow the documentation to associate the schema to `**/quickstart.yml`
 
-=== Previewing the quick start locally
+=== Previewing the quick starts locally
 
-The quick start catalog and, the quick start pages can be previewed locally.
+You can preview the quick start catalog and the quick start pages locally. (Note that the preview shows _all_ of the quick starts in the `app-services-guides` repository, not just one quickstart.)
```
Comment (bhardesty) [inline CONTRIBUTING.adoc:112]:
```suggestion
You can preview the quick start catalog and the quick start pages locally. (Note that the preview shows _all_ of the quick starts in the `app-services-guides` repository, not just one quick start.)
```

File: CONTRIBUTING.adoc [hunk 3]
```diff
@@ -130,16 +130,18 @@ npm install
...
+(Optional) If you want to run the local preview with custom attributes you can do that by providing an attributes file on the command line. The attributes file can be either an AsciiDoc file (only the attributes declared in the document will be used) or a YAML file.
++
 ----
 export ATTRIBUTES_FILE=<path to attributes file>
 npm run start:dev
 ----
 
+Your web browser should open automatically to http://localhost:9001 and show the quick start catalog.
+
+"Hot reload" is enabled which means that saving a change to any `quickstart.yml` or `.adoc` file will trigger a rebuild and reload the content in the browser automatically.
```
Comment (bhardesty) [inline CONTRIBUTING.adoc:142]:
```suggestion
"Hot reload" is enabled which means that saving a change to any `quickstart.yml` or `*.adoc` file will trigger a rebuild and reload the content in the browser automatically.
```

**Review by bhardesty — APPROVED** [review]
Thanks @MelissaFlinn! Just a couple really minor comments.

File: CONTRIBUTING.adoc [hunk 4]
```diff
@@ -130,16 +130,18 @@ npm install
 npm run start:dev
 ----
 +
-. Your web browser should open automatically to http://localhost:9001 and show the quick start catalog
-. Hot reload is enabled meaning saving a change to any `quickstart.yml` or `.adoc` file will trigger a rebuild and reload the content in the browser automatically
-
-If you want to run the local preview with custom attributes you can do that by providing an attributes file on the command line. The attributes file can be either an AsciiDoc file (only the attributes declared in the document will be used), or a YAML file.
-
+(Optional) If you want to run the local preview with custom attributes you can do that by providing an attributes file on the command line. The attributes file can be either an AsciiDoc file (only the attributes declared in the document will be used) or a YAML file.
```
Comment (bredamc) [inline CONTRIBUTING.adoc:133]:
```suggestion
(Optional) You can run the local preview with custom attributes by specifying an attributes file on the command line. The attributes file can be either an AsciiDoc file (only the attributes declared in the document are used) or a YAML file.
```

File: CONTRIBUTING.adoc [hunk 5]
```diff
@@ -130,16 +130,18 @@ npm install
...
++
 ----
 export ATTRIBUTES_FILE=<path to attributes file>
 npm run start:dev
 ----
 
+Your web browser should open automatically to http://localhost:9001 and show the quick start catalog.
+
+"Hot reload" is enabled which means that saving a change to any `quickstart.yml` or `*.adoc` file will trigger a rebuild and reload the content in the browser automatically.
```
Comment (bredamc) [inline CONTRIBUTING.adoc:142]:
```suggestion
"Hot reload" is enabled, which means that saving a change to any `quickstart.yml` or `*.adoc` file will trigger a rebuild and reload the content in the browser automatically.
```

**Review by bredamc — COMMENTED** [review]
Many thanks for updating the instructions, @MelissaFlinn -- just a few very minor comments from me :)

**Pull request merged**